src/pages/Agenda/Agenda.tsx

```diff
@@ -27,86 +36,111 @@ const Agenda: React.FC = () => {
     ));
   };
 
+  function filterSessionsByDay() {
+    const agendaItemsByDayArray = new Array<JSX.Element[]>(
+      Array.from(DAYS_MAP.keys()).length
+    );
+    for (let i = 0; i < agendaItemsByDayArray.length; i++) {
+      agendaItemsByDayArray[i] = new Array<JSX.Element>();
+    }
+    for (let i = 0; i < sessions.length; i++) {
+      const slotStart = sessions[i].slot_start;
+      if (slotStart) {
+        const day = new Date(slotStart).toDateString();
+        let dayIndex = -1;
+        switch (day) {
+          case DAYS_MAP.get("Day 1"):
+            dayIndex = 0;
+            break;
+          case DAYS_MAP.get("Day 2"):
+            dayIndex = 1;
+            break;
+          case DAYS_MAP.get("Day 3"):
+            dayIndex = 2;
+            break;
+          case DAYS_MAP.get("Day 4"):
+            dayIndex = 3;
+            break;
+          default:
+            console.log("unkown day: ", day);
+            break;
+        }
+        const sessionItem = sessions[i];
+        const shortTitle = shortenTitle(sessionItem.title);
```

Comment:
What is the reason to shorten the title? What is the difference from `text-overflow: ellipsis;`?